resources/views/blocks/_seachResults.antlers.html

```diff
@@ -0,0 +1,50 @@
+<div class="container mx-auto py-12 px-4 max-w-4xl">
+
+    <div class="mb-10 text-center">
+        <form action="{{ site:url }}/search" method="GET" class="flex justify-center items-center">
+            <input type="search" name="q" placeholder="Search" value="{{ get:q }}"
+                class="px-4 py-2 border-b border-minova-gold focus:outline-none text-xl w-1/2">
```

Comment:
- Change "px-4" to "pl-4"
- Change "py-2" to "py-4"
- Change "text-xl" to "text-2xl"